build/documentation/generate-typedocs.js

```diff
@@ -142,13 +154,13 @@ function createAPIFiles(packageName, srcPath) {
         }
     });
 
-    // sometimes important errors will get sent to stdout for this process
-    // leaving this here because it's useful for debugging, but generates a lot
-    // of output, makes the terminal output harder to read so it's commented out
-
-    // typedoc.stdout.on('data', (data) => {
-    //     console.error(`${data}`);
-    // });
+    // if set to true, will print detailed build output for typedoc process
+    // useful for debugging
+    if (verbose) {
+        typedoc.stdout.on('data', (data) => {
+            console.error(`${data}`);
```

Comment:
It seems a little strange to me to use `console.error` to log stdout but `console.log` to log errors - is there a reason for doing that?

Reply:
No, it's just me being inconsistent, I'll go in and change them all to one or the other